sdks/python/apache_beam/testing/load_tests/sideinput_test.py

```diff
@@ -27,11 +27,7 @@
   * side_input_type (str) - Required. Specifies how the side input will be
     materialized in ParDo operation. Choose from (dict, iter, list).
   * window_count (int) - The number of fixed sized windows to subdivide the
-    side input into. By default, no windows will be used.
-  * side_input_size (int) - The size of the side input. Must be equal to or
-    lower than the size of the main input. If lower, the side input will be
-    created by applying a :class:`beam.combiners.Sample
-    <apache_beam.transforms.combiners.Sample>` transform.
+    side input into. By default, a global window will be used.
```

Comment:
This should mention that by default the side_input will be subdivided into 1000 windows.

Reply:
Isn't it the other way around? Take a look at the constructor, where we have:
```python
self.get_option_or_default('window_count', default=1)
```
This applies both to the main and the side inputs.

Reply:
Yes, I confused number of windows with the number of initial elements from the SyntheticSDFAsSource. Thanks!